source/stable-release_iot/stable-release_iot.rst

```diff
@@ -6,13 +6,13 @@ Stable Releases IoT
 Stable Releases (IoT) is an Internet of Things (IoT) overlay on top of |C|. The
 Stable Releases (IoT) release provides users with an extensive set of features on
 Comet Lake (CML), Tiger Lake (TGL), and Elkhart Lake (EHL) platforms with a single
-Baseline/Image.
+Baseline/Image of Andoird 11 and Alder Lake (ADL) platforms with a single Baseline/Image of Andoird 12
```

Comment:
Stable Releases (IoT) release provides users with an extensive set of features on
Android 11: Comet Lake (CML), Tiger Lake (TGL), and Elkhart Lake (EHL) platforms with a single Baseline/Image. 
Android 12: Alder Lake (ADL) platforms with a single Baseline/Image.